web/src/components/tasks/LicenseStatusReceipt.tsx

```diff
@@ -61,16 +61,47 @@ const LicenseStatusLookup: Record<LicenseStatus, string> = {
   VOLUNTARY_SURRENDER: Config.licenseSearchTask.voluntarySurrenderPermitStatusText,
   WITHDRAWN: Config.licenseSearchTask.withdrawnPermitStatusText,
   UNKNOWN: "",
+  DRAFT: Config.licenseSearchTask.draftStatusText,
+  SUBMITTED: Config.licenseSearchTask.submittedStatusText,
+  UNDER_INTERNAL_REVIEW: Config.licenseSearchTask.underInternalReviewStatusText,
+  SPECIAL_REVIEW: Config.licenseSearchTask.specialReviewStatusText,
+  PENDING_DEFICIENCIES: Config.licenseSearchTask.pendingDeficienciesStatusText,
+  DEFICIENCIES_SUBMITTED: Config.licenseSearchTask.deficienciesSubmittedStatusText,
+  CHECKLIST_COMPLETED: Config.licenseSearchTask.checklistCompletedStatusText,
+  APPROVED: Config.licenseSearchTask.approvedStatusText,
+  PENDING_RENEWAL: Config.licenseSearchTask.pendingRenewalStatusText,
+  PENDING_REINSTATEMENT: Config.licenseSearchTask.pendingReinstatementStatusText,
+  INACTIVE: Config.licenseSearchTask.inactiveStatusText,
+  ABANDONED: Config.licenseSearchTask.abandonedStatusText,
+  SUSPENDED: Config.licenseSearchTask.suspendedStatusText,
+  REVOKED: Config.licenseSearchTask.revokedStatusText,
 };
 
 export const LicenseStatusReceipt = (props: Props): ReactElement => {
   const [theme, setTheme] = useState<PermitTheme>(pendingPermitTheme);
   const { business } = useUserData();
 
+  const isPending = (licenseStatus: LicenseStatus): boolean => {
+    const pendingStatus: LicenseStatus[] = [
+      "PENDING",
```

Comment:
[pebble] Can we add a test for this? To test for something other than "PENDING"?